ci-operator/jobs/stackrox/stackrox/stackrox-stackrox-master-postsubmits.yaml

```
@@ -787,11 +787,11 @@ postsubmits:
     decorate: true
     decoration_config:
       skip_cloning: true
-      timeout: 5h0m0s
+      timeout: 2h0m0s
     labels:
       ci.openshift.io/generator: prowgen
-    max_concurrency: 5
-    name: branch-ci-stackrox-stackrox-master-merge-openshift-4-qa-e2e-tests
+    max_concurrency: 1
```

You might want to bump this or the tests will stack up on merges.
```suggestion
    max_concurrency: 5
```

---

https://issues.redhat.com/browse/ROX-12039